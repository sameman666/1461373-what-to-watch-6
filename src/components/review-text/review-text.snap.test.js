import React from 'react';
import {render} from '@testing-library/react';
import ReviewText from './review-text';

it(`Should ReviewText block render correctly`, () => {
  const {container} = render(<ReviewText />);
  expect(container).toMatchSnapshot();
});
