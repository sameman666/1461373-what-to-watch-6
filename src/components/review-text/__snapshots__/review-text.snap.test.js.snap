// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewText block render correctly 1`] = `
<div>
  <textarea
    class="add-review__textarea"
    id="review"
    maxlength="400"
    name="review"
    placeholder="Review text"
  />
</div>
`;
